tetris.py

```diff
@@ -120,28 +120,60 @@ def new_block(self):
         self.current_block
         """
         if self.upcoming_block_shape is None:
-            self.current_block_shape = random.choice(shape_names)
+            self.current_block_shape = "L_rev"
         else:
             self.current_block_shape = self.upcoming_block_shape
-        self.current_block_center = (int(game_width / 2), -2)
+        self.current_block_center = (int(game_width / 2), 8)
         self.upcoming_block_shape = random.choice(shape_names)
+        self.index = 0
 
     def get_current_block(self):
         (x, y) = self.current_block_center
         if self.current_block_shape == "I":
-            return [(x - 2, y), (x - 1, y), (x, y), (x + 1, y)]
+            I = [
+                [(x - 2, y), (x - 1, y), (x, y), (x + 1, y)],
+                [(x, y - 3), (x, y - 2), (x, y - 1), (x, y)],
```

Comment:
I guess it would make sense to keep the center `(x, y)` exactly as it is, and only change other coords:

```suggestion
                [(x, y - 2), (x, y - 1), (x, y), (x, y + 1)],
```

Please change other blocks similarly, so that if one block contains `(x, y)`, then they all contain `(x, y)` in the same place.